src/NuGet.Clients/NuGet.PackageManagement.UI/Resources/Brushes.cs

```diff
@@ -149,7 +156,21 @@ public static class Brushes
  
         public static object ButtonBorderFocusedStyleBrushKey { get; private set; } = SystemColors.HighlightTextBrushKey; 
  
-        public static object ButtonFocusedStyleBrushKey { get; private set; } = SystemColors.HighlightTextBrushKey; 
+        public static object ButtonFocusedStyleBrushKey { get; private set; } = SystemColors.HighlightTextBrushKey;
+
+        public static object TabInactiveBrushKey { get; private set; } = SystemColors.InactiveCaptionBrushKey;
+
+        public static object TabInactiveTextBrushKey { get; private set; } = SystemColors.InactiveCaptionTextBrushKey;
```

Comment:
TabInactiveTextBrushKey also isn't used anywhere

Reply:
The same as TabInactiveBrushKey